test/MOI_wrapper.jl

```diff
@@ -1,82 +1,129 @@
+        r"test_constraint_PrimalStart_DualStart_SecondOrderCone$",
+        # Expression: MOI.get(model, MOI.TerminationStatus()) == MOI.INFEASIBLE
+        #  Evaluated: MathOptInterface.INFEASIBLE_OR_UNBOUNDED == MathOptInterface.INFEASIBLE
+        r"test_conic_NormInfinityCone_INFEASIBLE$",
+        r"test_conic_NormOneCone_INFEASIBLE$",
+        r"test_solve_DualStatus_INFEASIBILITY_CERTIFICATE_Interval_lower$",
+        r"test_solve_DualStatus_INFEASIBILITY_CERTIFICATE_Interval_upper$",
+        # Incorrect objective
+        # See https://github.com/jump-dev/MathOptInterface.jl/issues/1759
+        r"test_unbounded_MIN_SENSE$",
+        r"test_unbounded_MIN_SENSE_offset$",
+        r"test_unbounded_MAX_SENSE$",
+        r"test_unbounded_MAX_SENSE_offset$",
+        r"test_infeasible_MAX_SENSE$",
+        r"test_infeasible_MAX_SENSE_offset$",
+        r"test_infeasible_MIN_SENSE$",
+        r"test_infeasible_MIN_SENSE_offset$",
+        r"test_infeasible_affine_MAX_SENSE$",
+        r"test_infeasible_affine_MAX_SENSE_offset$",
+        r"test_infeasible_affine_MIN_SENSE$",
+        r"test_infeasible_affine_MIN_SENSE_offset$",
+        # TODO remove when PR merged
+        # See https://github.com/jump-dev/MathOptInterface.jl/pull/1769
+        r"test_objective_ObjectiveFunction_blank$",
+        # FIXME investigate
+        #  Expression: isapprox(MOI.get(model, MOI.ObjectiveValue()), T(2), config)
+        #   Evaluated: isapprox(5.999999984012059, 2.0, ...
+        r"test_modification_delete_variables_in_a_batch$",
+        # FIXME investigate
+        #  Expression: isapprox(MOI.get(model, MOI.ObjectiveValue()), objective_value, config)
+        #   Evaluated: isapprox(-2.1881334077988868e-7, 5.0, ...
+        r"test_objective_qp_ObjectiveFunction_edge_case$",
+        # FIXME investigate
+        #  Expression: isapprox(MOI.get(model, MOI.ObjectiveValue()), objective_value, config)
+        #   Evaluated: isapprox(-2.1881334077988868e-7, 5.0, ...
+        r"test_objective_qp_ObjectiveFunction_zero_ofdiag$",
+        # FIXME investigate
+        #  Expression: isapprox(MOI.get(model, MOI.ConstraintPrimal(), index), solution_value, config)
+        #   Evaluated: isapprox(2.5058846553349667e-8, 1.0, ...
+        r"test_variable_solve_with_lowerbound$",
+        # FIXME investigate
+        # See https://github.com/jump-dev/SDPA.jl/runs/7246518765?check_suite_focus=true#step:6:128
+        # Expression: ≈(MOI.get(model, MOI.ConstraintDual(), c), T[1, 0, 0, -1, 1, 0, -1, -1, 1] / T(3), config)
+        #  Evaluated: ≈([0.3333333625488728, -0.16666659692134123, -0.16666659693012292, -0.16666659692134123, 0.33333336253987234, -0.16666659692112254, -0.16666659693012292, -0.16666659692112254, 0.333333362548654], [0.3333333333333333, 0.0, 0.0, -0.3333333333333333, 0.3333333333333333, 0.0, -0.3333333333333333, -0.3333333333333333, 0.3333333333333333]
+        r"test_conic_PositiveSemidefiniteConeSquare_3$",
+        # FIXME
```

Comment:
are these copied from SDPA.jl? Some of them might pass here with the presolve (because we've found that is necessary to function correctly in some cases)

Reply:
Yes but I have not updated the presolve code yet. It should be udpated to work with `MOI.FileFormats.SDPA.Model`. Maybe https://www.google.com/search?client=firefox-b-d&q=MathOptPresolve.jl could be used ? I haven't checked in details

Reply:
I would like to take a look at https://github.com/mtanneau/MathOptPresolve.jl so to see if it has everything needed for convex problems and then recommend people to always try using this layer (in addition to the Dualization.jl layer which should always be tried for conic optimization).